apps/custom-ui-app-docusign/public/js/settings.js

```diff
@@ -0,0 +1,31 @@
+// Add settings to the database upon save
+$(document).on('click', '#save_configs', function () {
+    const settings = {
+            docusign_token: $("#docusign_token").val(),
+            docusign_accid: $("#docusign_accid").val()
+        },
+        company_id = $("#company_id").val();
+    $.ajax({
```

Comment:
Can you use Fetch instead of ajax? https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API it's built into the browser